src/platform/site-wide/mega-menu/constants/MY_VA_LINK.js

```diff
@@ -1,5 +1,5 @@
 export default {
   className: 'my-va-top-nav',
-  href: 'https://www.va.gov/my-va/',
+  href: '/my-va/',
```

Comment:
This link constant is only shown for authenticated users on va.gov, so we should omit the protocol and hostname, as the browser will supply this information.